lib/fluent/plugin/out_slack.rb

```diff
@@ -144,24 +148,34 @@ def configure(conf)
           log.warn "out_slack: `as_user` parameter are not available for Incoming Webhook"
         end
         @slack = Fluent::SlackClient::IncomingWebhook.new(@webhook_url)
-      elsif @slackbot_url
```

Comment:
I wonder whether I should split this validation code to some validation method, or not. 🤔 
And I decided to just add one if-clause nest and validation on`slackbot_url` case and `token` case for channel config.